code/lib/builder-vite/src/build.ts

```diff
@@ -8,11 +8,17 @@ export async function build(options: ExtendedOptions) {
   const { presets } = options;
 
   const config = await commonConfig(options, 'build');
-  config.build = {
-    outDir: options.outputDir,
-    emptyOutDir: false, // do not clean before running Vite build - Storybook has already added assets in there!
-    sourcemap: true,
-  };
+  config.build = mergeConfig(config, {
```

Comment:
We could remove the `mergeConfig` here, since we do not want to merge the build config with the user's build config.